specification/api/components/endpoints/listWhitemail.yml

```diff
@@ -0,0 +1,14 @@
+summary: List batches of whitemail letters
```

Comment:
We've agreed with Gaby that whitemail will be out of scope for this first version of the API, so we can remove the endpoints.